src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,409 @@
+    term2 *= c_ts[num_stages - 1 - i] * 2 / 3 # 2 / 3 = b_S
+
+    c_eq[i] = monomial_coeffs[i] - term2
+    # Third-order consistency condition (Cf. eq. (27) from https://doi.org/10.1016/j.jcp.2022.111470
+    c_eq[num_stage_evals - 2] = 1 - 4 * a_coeff[num_stage_evals] -
+                                a_coeff[num_stage_evals - 1]
+
+    return c_eq
+end
+
+# Compute the Butcher tableau for a paired explicit Runge-Kutta method order 3
+# using a list of eigenvalues
+function compute_PairedExplicitRK3_butcher_tableau(num_stages, tspan,
+                                                   eig_vals::Vector{ComplexF64};
+                                                   verbose = false, cS2)
+    # Initialize array of c
+    c = compute_c_coeffs_SSP33(num_stages, cS2)
+
+    # Initialize the array of our solution
+    a_unknown = zeros(num_stages - 2)
+
+    # Special case of e = 3
+    if num_stages == 3
+        #TODO: with this new defining of what we are solving from nlsolve, this array shouldn't be called a_unknown anymore
+        # or maybe it should be called a_unknown but only with one member
+        a_unknown = [0, c[2], 0.25]
+        dt_opt = 42.0 # TODO! This is a placeholder value
+    else
+        # Calculate coefficients of the stability polynomial in monomial form
+        consistency_order = 3
+        dtmax = tspan[2] - tspan[1]
+        dteps = 1e-9
+
+        num_eig_vals, eig_vals = filter_eig_vals(eig_vals; verbose)
+
+        monomial_coeffs, dt_opt = bisect_stability_polynomial(consistency_order,
+                                                              num_eig_vals, num_stages,
+                                                              dtmax,
+                                                              dteps,
+                                                              eig_vals; verbose)
+        monomial_coeffs = undo_normalization!(monomial_coeffs, consistency_order,
+                                              num_stages)
+
+        # Solve the nonlinear system of equations from monomial coefficient and
+        # Butcher array abscissae c to find Butcher matrix A
+        # This function is extended in TrixiNLsolveExt.jl
+        a_unknown = solve_a_unknown!(a_unknown, num_stages, monomial_coeffs, cS2, c;
+                                     verbose)
+    end
+    # Fill A-matrix in P-ERK style
+    a_matrix = zeros(num_stages - 2, 2)
+    a_matrix[:, 1] = c[3:end]
+    a_matrix[:, 1] -= a_unknown
+    a_matrix[:, 2] = a_unknown
+
+    return a_matrix, c
+end
+
+# Compute the Butcher tableau for a paired explicit Runge-Kutta method order 3
+# using provided values of coefficients a in A-matrix of Butcher tableau
+function compute_PairedExplicitRK3_butcher_tableau(num_stages,
+                                                   base_path_a_coeffs::AbstractString;
+                                                   cS2)
+
+    # Initialize array of c
+    c = compute_c_coeffs_SSP33(num_stages, cS2)
```

Comment:
Hm, maybe we should not have `SSP33` in the function name to avoid confusion.The comments in lines 14 and 15 should suffice.